projects/optic/src/commands/oas/verify.ts

```diff
@@ -188,6 +216,38 @@ export async function verifyCommand(): Promise<Command> {
         },
       });
 
+      if (options.upload) {
+        if (!opticUrlDetails) {
+          console.error(
+            `File ${specPath} does not have an optic url. Files must be added to Optic and have an x-optic-url key before verification data can be uploaded.`
+          );
+          console.error(
+            `${chalk.yellow('Hint: ')} Run optic api add ${specPath}`
+          );
+          process.exitCode = 1;
+          return;
+        }
+
+        const { orgId, apiId } = opticUrlDetails;
+        const tags: string[] = [];
+        if (config.vcs?.type === VCS.Git) {
+          tags.push(`git:${config.vcs.sha}`);
+          const currentBranch = await Git.getCurrentBranchName();
+          tags.push(sanitizeGitTag(`gitbranch:${currentBranch}`));
```

Comment:
Do you think it's worth adding `--tag` here? I don't think we'll run into the detached head state in verify if it's used locally. 

And for now, we can only really run this locally (i.e. there isn't a good way to run this in CI b/c of captures)